### DISCUSSION TITLE
Fuzzy lookup failed

### DISCUSSION THREAD

**Discussion by hongjie123** [ckan/ckan#6438] [Q&A]
sudo apt install -y solr-tomcat (tomcat9)
CKAN configuration file (/etc/ckan/default/ckan.ini),solr_url=http://127.0.0.1:8983/solr
![image](https://user-images.githubusercontent.com/54384385/134650710-12946e9d-495a-479c-a40b-16cbaff2098a.png)
![image](https://user-images.githubusercontent.com/54384385/134650920-a6b8f345-d4b9-4abe-8711-25db63e7fc47.png)
Keyword search failed, complete search succeeded.

**Comment by hongjie123** [comment]
Other questions:
1.The email function is unsuccessful.
![image](https://user-images.githubusercontent.com/54384385/134651945-c7f1971f-6a89-4fa2-833d-cbd1fbbe9ec1.png)
Wait a few minutes for a problem:
![image](https://user-images.githubusercontent.com/54384385/134652216-2c3d6ca1-d531-4263-a92a-5df2a61e960c.png)
I didn't get any mail.

2.Failed to add a new user to the organization.
![image](https://user-images.githubusercontent.com/54384385/134653827-9e3c0e55-d12e-4aff-97cb-98d6667f8c89.png)
Wait a few minutes for a problem:
![image](https://user-images.githubusercontent.com/54384385/134653612-78fbca6b-f20e-4fe6-815b-753345d0010b.png)
I didn't get any mail.

**Comment by amercader** [comment]
For the first issue, it looks like you need to customize your Solr schema to support this kind of searches. 
For the second one, both issues seem to be related to the email server not being properly configured to send emails
